libs/tracing-impl/src/main/kotlin/net/corda/tracing/brave/BraveTracingService.kt

```diff
@@ -111,9 +111,9 @@ internal class BraveTracingService(serviceName: String, zipkinHost: String?, sam
     }
 
     private val serverSampler: SamplerFunction<HttpRequest> = HttpRuleSampler.newBuilder()
-        .putRule(and(methodEquals("POST"), pathStartsWith("/api/v5_1/flow")), sampler(samplesPerSecond))
-        .putRule(and(methodEquals("POST"), pathStartsWith("/api/v1/flow")), sampler(samplesPerSecond))
-        .putRule(pathStartsWith("/"), sampler(samplesPerSecond)).build()
+        .putRule(and(methodEquals("GET"), pathStartsWith("/metrics")), Sampler.NEVER_SAMPLE) // Disable tracing for the specified path
```

Comment:
I think we should skip checking the method and just check path because we don't want to trace any requests on this path.

```suggestion
        .putRule(pathStartsWith("/metrics"), Sampler.NEVER_SAMPLE) // Disable tracing for the specified path
```